compiler/resolution/functionResolution.cpp

```diff
@@ -5554,6 +5579,7 @@ resolveCoerce(CallExpr* call) {
 //
 static bool backendRequiresCopyForIn(Type* t) {
   return (isRecord(t) ||
+          isUnion(t) ||
```

Comment:
More of an observation than a critique of this PR: I wonder if we should revisit this logic as Qualifiers become more widely used. I'm guessing the result of this function may be different in the _ref type world.